nixos/modules/virtualisation/digital-ocean-init.nix

```diff
@@ -0,0 +1,62 @@
+{ config, pkgs, lib, ... }:
+with lib;
+{
+  options.virtualisation.digitalOcean.rebuildFromUserData = mkOption {
+    type = bool;
+    enable = true;
+    example = true;
+    description = "Whether to reconfigure the system from Digital Ocean user data";
+  };
+
+  config = {
+    systemd.services.digital-ocean-init = {
+      description = "Reconfigure the system from Digital Ocean uesrdata on startup";
+      wantedBy = [ "multi-user.target" ];
+      unitConfig = {
+        After = [ "network-online.target" ];
```

Comment:
Perhaps we could add a

`ConditionFileExists=!/etc/nixos/configuration.nix`

So that this only runs on first boot? Same as we do for SSH keys.

User data cannot be changed after droplet start anyway and otherwise user changes will get overriden during boot

Reply:
Done